build.gradle

```diff
@@ -13,7 +13,7 @@ allprojects {
             java {
                 toolchain {
                     languageVersion = JavaLanguageVersion.of(javaVersion)
-                    vendor = JvmVendorSpec.matching('Eclipse Adoptium')
+//                    vendor = JvmVendorSpec.matching('Eclipse Adoptium')
```

Comment:
Please remove this comment and keep it locally if it doesn't work on your machine

Reply:
This setting is not critical, but may cause problems for users.